pkg/controller/configmap/configmap_controller.go

```diff
@@ -133,7 +137,7 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 	// Fetch ncp-install CRD instance
 	ncpInstallCrd := &operatorv1.NcpInstall{}
 	err := r.client.Get(context.TODO(), types.NamespacedName{Name: operatortypes.NcpInstallCRDName,
-		Namespace: operatortypes.OperatorNamespace}, ncpInstallCrd)
+		Namespace: r.status.OperatorNamespace}, ncpInstallCrd)
```

Comment:
Same here, if WATCH_NAMESPACE is set as "", do we need to use "nsx-system-operator" to get resources?

Reply:
yup

Reply:
I modified the code ensured everything works properly if the operator is deployed in `nsx-system` namespace